machine_learning_hep/data/data_prod_20200304/database_ml_parameters_D0pp_0304.yml

```diff
@@ -145,9 +145,9 @@ D0pp:
       fracmerge: [0.08,0.08,0.08] #list of periods
       seedmerge: [12,12,12] #list of periods
       period: [LHC16pp,LHC17pp,LHC18pp] #list of periods
-      unmerged_tree_dir: [/data/TTree/D0DsLckAnywithJets/vAN-20200304_ROOT6-1/pp_2016_data/374_20200304-2028/merged,
-                          /data/TTree/D0DsLckAnywithJets/vAN-20200304_ROOT6-1/pp_2017_data/375_20200304-2028/merged,
-                          /data/TTree/D0DsLckAnywithJets/vAN-20200304_ROOT6-1/pp_2018_data/376_20200304-2028/merged] #list of periods
+      unmerged_tree_dir: [/tmp/TTree/D0DsLckAnywithJets/vAN-20200304_ROOT6-1/pp_2016_data/374_20200304-2028/merged,
```

Comment:
We should not set the paths in the default database to a value different different from what it was before. Otherwise, people might not find their data in the future.
In general, changing paths in the databases in a PR can easily cause confusion. 
If really needed, we should make a validation database

Comment applies to many other lines